roles/satosa/tasks/main.yml

```diff
@@ -235,10 +235,13 @@
     - name: Create metadata directory
       file: path="{{ satosa_env_dir }}/metadata" state=directory mode=0755
 
+    - name: Create clients.d directory
+      file: path="{{ satosa_env_dir }}/metadata/clients.d" state=directory mode=0755
+
     - name: Copy clients.json example file
       template:
         src: clients.json.j2
-        dest: "{{ satosa_env_dir }}/metadata/clients.json"
+        dest: "{{ satosa_env_dir }}/metadata/clients.d/default.json"
```

Comment:
Kan je de directorynaam iets beschrijvender maken?  Dus bv "{{ satosa_env_dir }}/oidc_clients.d" ?  Als het kan liever niet in dezelfde dir waar ook saml metadata staat.